File: cite de la culture feres/CiteDeLaCulture/src/Entite/Speaker.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entite;

/**
 *
 * @author Debagi
 */
public class Speaker {
    private int id_speaker ;
  private String nom_speaker,prenom_speaker,mail_speaker,date_arrive,date_depart,description_speaker;

    public Speaker(int id_speaker, String nom_speaker, String prenom_speaker, String mail_speaker, String date_arrive, String date_depart, String description_speaker) {
        this.id_speaker = id_speaker;
        this.nom_speaker = nom_speaker;
        this.prenom_speaker = prenom_speaker;
        this.mail_speaker = mail_speaker;
        this.date_arrive = date_arrive;
        this.date_depart = date_depart;
        this.description_speaker = description_speaker;
    }

    public Speaker() {
    }

    public int getId_speaker() {
        return id_speaker;
    }

    public void setId_speaker(int id_speaker) {
        this.id_speaker = id_speaker;
    }

    public String getNom_speaker() {
        return nom_speaker;
    }

    public void setNom_speaker(String nom_speaker) {
        this.nom_speaker = nom_speaker;
    }

    public String getPrenom_speaker() {
        return prenom_speaker;
    }

    public void setPrenom_speaker(String prenom_speaker) {
        this.prenom_speaker = prenom_speaker;
    }

    public String getMail_speaker() {
        return mail_speaker;
    }

    public void setMail_speaker(String mail_speaker) {
        this.mail_speaker = mail_speaker;
    }

    public String getDate_arrive() {
        return date_arrive;
    }

    public void setDate_arrive(String date_arrive) {
        this.date_arrive = date_arrive;
    }

    public String getDate_depart() {
        return date_depart;
    }

    public void setDate_depart(String date_depart) {
        this.date_depart = date_depart;
    }

    public String getDescription_speaker() {
        return description_speaker;
    }

    public void setDescription_speaker(String description_speaker) {
        this.description_speaker = description_speaker;
    }

    @Override
    public String toString() {
        return "Speaker{" + "id_speaker=" + id_speaker + ", nom_speaker=" + nom_speaker + ", prenom_speaker=" + prenom_speaker + ", mail_speaker=" + mail_speaker + ", date_arrive=" + date_arrive + ", date_depart=" + date_depart + ", description_speaker=" + description_speaker + '}';
    }
  
}

File: Elyes/src/CiteBundle/Entity/Speaker.php
<?php

namespace CiteBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Speaker
 *
 * @ORM\Table(name="speaker", indexes={@ORM\Index(name="idU_fk", columns={"iduser"})})
 * @ORM\Entity
 */
class Speaker
{
    /**
     * @var integer
     *
     * @ORM\Column(name="idspeaker", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idspeaker;

    /**
     * @var string
     *
     * @ORM\Column(name="nomspeaker", type="string", length=255, nullable=true)
     */
    private $nomspeaker;

    /**
     * @var string
     *
     * @ORM\Column(name="prenomspeaker", type="string", length=255, nullable=true)
     */
    private $prenomspeaker;

    /**
     * @var string
     *
     * @ORM\Column(name="mailspeaker", type="string", length=255, nullable=true)
     */
    private $mailspeaker;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="datearrive", type="date", nullable=true)
     */
    private $datearrive;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="datedepart", type="date", nullable=true)
     */
    private $datedepart;

    /**
     * @var string
     *
     * @ORM\Column(name="descriptionspeaker", type="string", length=255, nullable=true)
     */
    private $descriptionspeaker;

    /**
     * @var string
     *
     * @ORM\Column(name="PhotoSpeaker", type="string", length=255, nullable=true)
     */
    private $photospeaker;

    /**
     * @var \User
     *
     * @ORM\ManyToOne(targetEntity="User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="iduser", referencedColumnName="iduser")
     * })
     */
    private $iduser;

    /**
     * @var string
     *
     * @ORM\Column(name="image_speaker", type="string", length=255, nullable=true)
     */
    private $image;




    /**
     * @return int
     */
    public function getIdspeaker()
    {
        return $this->idspeaker;
    }

    /**
     * @param int $idspeaker
     */
    public function setIdspeaker($idspeaker)
    {
        $this->idspeaker = $idspeaker;
    }

    /**
     * @return string
     */
    public function getNomspeaker()
    {
        return $this->nomspeaker;
    }

    /**
     * @param string $nomspeaker
     */
    public function setNomspeaker($nomspeaker)
    {
        $this->nomspeaker = $nomspeaker;
    }

    /**
     * @return string
     */
    public function getPrenomspeaker()
    {
        return $this->prenomspeaker;
    }

    /**
     * @param string $prenomspeaker
     */
    public function setPrenomspeaker($prenomspeaker)
    {
        $this->prenomspeaker = $prenomspeaker;
    }

    /**
     * @return string
     */
    public function getMailspeaker()
    {
        return $this->mailspeaker;
    }

    /**
     * @param string $mailspeaker
     */
    public function setMailspeaker($mailspeaker)
    {
        $this->mailspeaker = $mailspeaker;
    }

    /**
     * @return \DateTime
     */
    public function getDatearrive()
    {
        return $this->datearrive;
    }

    /**
     * @param \DateTime $datearrive
     */
    public function setDatearrive($datearrive)
    {
        $this->datearrive = $datearrive;
    }

    /**
     * @return \DateTime
     */
    public function getDatedepart()
    {
        return $this->datedepart;
    }

    /**
     * @param \DateTime $datedepart
     */
    public function setDatedepart($datedepart)
    {
        $this->datedepart = $datedepart;
    }

    /**
     * @return string
     */
    public function getDescriptionspeaker()
    {
        return $this->descriptionspeaker;
    }

    /**
     * @param string $descriptionspeaker
     */
    public function setDescriptionspeaker($descriptionspeaker)
    {
        $this->descriptionspeaker = $descriptionspeaker;
    }

    /**
     * @return string
     */
    public function getPhotospeaker()
    {
        return $this->photospeaker;
    }

    /**
     * @param string $photospeaker
     */
    public function setPhotospeaker($photospeaker)
    {
        $this->photospeaker = $photospeaker;
    }

    /**
     * @return \User
     */
    public function getIduser()
    {
        return $this->iduser;
    }

    /**
     * @param \User $iduser
     */
    public function setIduser($iduser)
    {
        $this->iduser = $iduser;
    }

    /**
     * @return string
     */
    public function getImage()
    {
        return $this->image;
    }

    /**
     * @param string $image
     */
    public function setImage($image)
    {
        $this->image = $image;
    }
}

